jicofo/rootfs/defaults/sip-communicator.properties

```diff
@@ -1,5 +1,6 @@
 org.jitsi.jicofo.ALWAYS_TRUST_MODE_ENABLED=true
 org.jitsi.jicofo.BRIDGE_MUC={{ .Env.JVB_BREWERY_MUC }}@{{ .Env.XMPP_INTERNAL_MUC_DOMAIN }}
+org.jitsi.jicofo.XMPP_MUC_COMPONENT_PREFIX=muc
```

Comment:
Let's not hardcode it, since it can be changed by the user: https://github.com/jitsi/docker-jitsi-meet/blob/2a0120de8b4b274bcb22c98426705c54e570edd3/env.example#L190

You should use frep to get the first element in that domain.